opentreemap/treemap/js/src/MapManager.js

```diff
@@ -8,7 +8,9 @@ var $ = require('jquery'),
     makeLayerFilterable = require('treemap/makeLayerFilterable'),
     urlState = require('treemap/urlState'),
 
-    _ZOOM_OPTIONS = {maxZoom: 21};
+    MAX_ZOOM_OPTION = {maxZoom: 21},
```

Comment:
why drop the underscore? doesn't look like these have changed from internal to exported.